src/main/java/seedu/duke/StonksXD.java

```diff
@@ -17,6 +22,7 @@ public StonksXD() {
     }
 
     public void run() {
+        dataManager.load(parser, finances, ui);
```

Comment:
Would it be better to put this line inside the constructor instead?

Reply:
I feel that it is fine currently. I am also not too sure how to implement your method, maybe you can try first?